src/requirements.in

```diff
@@ -23,22 +23,22 @@ python-keystoneclient == 4.0.0
 python-swiftclient == 3.9.0
 environs == 7.3.1
 flask_bcrypt == 0.7.1
-docker == 4.2.0
+docker == 5.0.0
```

Comment:
Gaat dit zeker goed? major versie bump, betekend dat API/functies veranderd?

Reply:
De breaking change die erin zit is "Remove support for Python 2.7". Lokaal heb ik een aantal DAGs getest, blijven gewoon werken. Maar, ik ga extra testen als het op ACC staat. Want, ook cx_oracle heeft major versie bump. Lijkt ook alleen om python2.7 removal te gaan.